ribs/archives/_cvt_archive.py

```diff
@@ -260,9 +243,12 @@ def index_of(self, measures_batch):
                 self._centroid_kd_tree.query(measures_batch))[1].astype(
                     np.int32)
 
-        # Older versions of Numba do not support the axis argument for argmin,
-        # so the argmin is done here instead of in `_brute_force_nn_numba`.
-        return np.argmin(
-            self._brute_force_nn_numba(measures_batch, self.centroids),
-            axis=1,
-        ).astype(np.int32)
+        # Brute force distance calculation -- start by taking the difference
```

Comment:
Note: remove brute force implementation in the future.

Reply:
Let's keep it for legacy purposes -- nice to have a default implementation.